Allow runtime build without tracy tools but with instrumented.

stellaraccident opened the pull request:
Only enables tracy tools on Linux (disable on MacOS/Windows where we don't have deps).

powderluv commented on build_tools/python_deploy/build_macos_packages.sh:
```diff
@@ -76,6 +76,7 @@ function run() {
 
 function build_iree_runtime() {
   # TODO(antiagainst): remove Vulkan once IREE_HAL_DRIVER_METAL is stable
+  export IREE_RUNTIME_BUILD_TRACY=ON
```
shouldn't this be `IREE_RUNTIME_BUILD_TRACY=ON \`

stellaraccident committed:
Allow runtime build without tracy tools but with instrumented.

Only enables tracy tools on Linux (disable on MacOS/Windows where we don't have deps).
